unidist/run.py

```diff
@@ -0,0 +1,247 @@
+            else:
+                raise RuntimeError(
+                    "`num_workers` and `hosts` parameters must have the equal number of values."
+                )
+
+        hosts = _validate_hosts(hosts)
+        num_workers = _validate_num_workers(num_workers)
+        workers_dir = "/tmp"
+        command = ["mpiexec", "-hosts"]
+
+        hosts_str = f"{_get_localhost_ip()}:1,{_get_localhost_ip()}:1,"
+        for host, n in zip(hosts, num_workers):
+            hosts_str += host + ":" + n + ","
+
+        command_executor = ["-n", "1", executor, script]
+        command_monitor = [
+            "-n",
+            "1",
+            "-wdir",
+            workers_dir,
+            "python3",
+            unidist_home_path + "/unidist/core/backends/mpi/core/monitor.py",
+        ]
+
+        def get_worker_command(num_workers):
+            return [
+                "-n",
+                num_workers,
+                "-wdir",
+                workers_dir,
+                "python3",
+                unidist_home_path + "/unidist/core/backends/mpi/core/worker.py",
+            ]
+
+        command += [hosts_str] + command_executor + [":"] + command_monitor
+
+        for n in num_workers:
+            command += [":"] + get_worker_command(n)
+    else:
+        command = [executor, script]
+
+    return command
+
+
+def main():
+    """Run unidist application."""
+    usage_examples = [
+        "\n\tIn case 'unidist' is installed, use binary:",
+        "\n\tunidist script.py  # Ray backend is used",
+        "\n\tunidist script.py --backend MPI  # MPI backend is used",
+        "\n\tunidist script.py --executor pytest -b Dask  # Dask backend is used, running using 'pytest'",
+        "\n\n\tTo run from sources use 'unidist/run.py':",
+        "\n\tpython3 unidist/run.py script.py -b MPI --num_workers=16 --hosts localhost  # MPI backend uses 16 workers on 'localhost' node",
+        "\n\tunidist script.py -b MPI --num_workers=2 4 --hosts localhost x.x.x.x  # MPI backend uses 2 workers on 'localhost' and 4 on 'x.x.x.x'",
+    ]
+    parser = argparse.ArgumentParser(
+        description="Run python code with 'unidist' under the hood.",
+        usage="".join(usage_examples),
+    )
+
+    required_args_group = parser.add_argument_group("required arguments")
+    mpi_args_group = parser.add_argument_group("MPI-backend specific arguments")
+    required_args_group.add_argument("script", help="set script to be run")
+    parser.add_argument(
+        "-b",
+        "--backend",
+        type=str,
+        choices=["Ray", "MPI", "Dask", "MultiProcessing", "Python"],
+        default="Ray",
+        help="set an execution backend. Default is 'Ray'",
+    )
+    parser.add_argument(
+        "-e",
+        "--executor",
+        type=str,
+        default="python3",
+        help="set an executable to run. Default is 'python3'",
+    )
+    mpi_args_group.add_argument(
+        "-num_workers",
+        "--num_workers",
```

Comment:
This should be renamed to `cpu_count` and can be used for other backends.

Reply:
Do you mean using `unidist.config.CpuCount`?  In that case I have the question https://github.com/modin-project/unidist/pull/14#discussion_r751052817

Reply:
Yes. See the answer below.

Reply:
`cpu_count` isn't a good name for CLI parameter. `num_cpus` will be more habitual naming for users, because most part of quantitative CLI parameters starts with `number*/num*/n*`.